jest.base.config.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  globals: {
+    'ts-jest': {
+      tsConfigFile: './src/tsconfig.spec.json',
+      ignoreCoverageForAllDecorators: true,
+    },
+    '__TRANSFORM_HTML__': true,
+  },
+  // transform: {'^.+\\.(ts|js|html)$': '<rootDir>/node_modules/jest-preset-angular/preprocessor.js' },
```

Comment:
Several pieces of commented code here?

Reply:
Per the previous comment, I don't know what's needed. No qualms about taking these all out.

Reply:
I'd prefer not committing commented code.